pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -286,36 +292,58 @@ type staticPodStateFunc func(ctx context.Context, nodeName string) (state static
 // - not updating
 // - ready
 // - at the revision claimed in CurrentRevision.
-func nodeToStartRevisionWith(ctx context.Context, getStaticPodStateFn staticPodStateFunc, nodes []operatorv1.NodeStatus) (int, string, error) {
-	if len(nodes) == 0 {
-		return 0, "", fmt.Errorf("nodes array cannot be empty")
+func nodeToStartRevisionWith(ctx context.Context, getStaticPodStateFn staticPodStateFunc, nodeStatuses []operatorv1.NodeStatus, nodes []*corev1.Node) (int, string, error) {
```

Comment:
could we document the order in which the nodes will be processed?

Reply:
done